src/minute/Snakefile

```diff
@@ -278,6 +289,19 @@ rule bowtie2:
         "| samtools sort -@ {threads} -o {output.bam} -"
 
 
+rule quality_filter_reads:
```

Comment:
This could be misinterpreted as being about filtering reads by sequencing quality. Perhaps `quality_filter_alignments` instead? Or `filter_by_mapping_quality`/`filter_by_mapq`.